dependency/dependency_test.go

```diff
@@ -239,6 +229,34 @@ func (c *ClientSet) createConsulTestResources() error {
 	return nil
 }
 
+func (c *ClientSet) createConsulSamenessGroups(name, partition, failoverPartition string) error {
+	members := append([]api.SamenessGroupMember{}, api.SamenessGroupMember{
+		Partition: partition,
+	}, api.SamenessGroupMember{
+		Partition: failoverPartition,
+	})
+	sg := &api.SamenessGroupConfigEntry{
+		Kind:               QuerySamenessGroup,
```

Comment:
nit: does the API package expose the config entry kind constants? If not I'd just use the literal string here rather than tie this to our query which just happens to use the same name right now

Reply:
good call.

Reply:
fixed in edd0f7394427f64c6fc9de7aa74dd6bac9c32f80